src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -116,6 +117,21 @@
         (assoc acc project-name (.getInputStream jar-file jar-entry)))
       acc)))
 
+(defn get-stream-from-project-jar
+  [lein-project file-path]
+  (let [project-jar-file (fs/file (:target-path lein-project)
+                                  (str
+                                   (:name lein-project)
+                                   "-"
+                                   (:version lein-project)
+                                   ".jar"))]
+    (if (fs/exists? project-jar-file)
+      (let [project-jar (JarFile. project-jar-file)]
+        (if-let [project-jar-entry (find-file-in-jar project-jar
+                                                     file-path)]
+          (.getInputStream project-jar project-jar-entry)))
+      (log/warnf "Unable to find project jar file: '%s'" project-jar-file))))
```

Comment:
Yes, the `build` sub task calls the `stage` subtask which in turn calls the method that is run for leiningen's `uberjar` task. I ran `lein ezbake stage` locally and saw that both the uberjar and the local project jar are created--I believe the `uberjar` task depends on the `jar` task.